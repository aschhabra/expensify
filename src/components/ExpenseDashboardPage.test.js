
import React from "react";
import {shallow} from "enzyme";
import ExpenseDashboardPage from "./ExpenseDashboardPage";
import expenses from "../fixtures/expenses"

test( "test should render ExpenseList with expenses",()=>{
  const wrapper= shallow(<ExpenseDashboardPage/>);


  expect(wrapper).toMatchSnapshot();
});
