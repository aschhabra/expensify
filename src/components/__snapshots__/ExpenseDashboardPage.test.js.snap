// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should render ExpenseList with expenses 1`] = `
<div>
  <p>
     This is form dashboard
  </p>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
